load/loader.go

```diff
@@ -344,3 +382,23 @@ func (l *CommonBenchmarkRunner) report(period time.Duration) {
 		prevTime = now
 	}
 }
+
+// Track tsbs config on prometheus pushing a single event in the start.
+func (c *CommonBenchmarkRunner) pushEventToPrometheus(event string) {
+	eventTime := prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: fmt.Sprintf("tsbs_load_%s", event),
+		Help: "TSBS load start/finish events.",
+	})
+	eventTime.SetToCurrentTime()
+	if err := push.New("http://pushgateway:9091", "tsbs_load").
```

Comment:
I'm not sure if we should keep this naming as pushgateway estatically or have a way to allow people to configure it.

Reply:
IMHO Configurable would be much better.